deprecated/admin/taxonomy/class-taxonomy-social-fields.php

```diff
@@ -0,0 +1,22 @@
+<?php
+/**
+ * WPSEO plugin file.
+ *
+ * @package WPSEO\Admin
+ */
+
+/**
+ * This class parses all the values for the social tab in the Yoast SEO settings metabox.
+ */
+class WPSEO_Taxonomy_Social_Fields extends WPSEO_Taxonomy_Fields {
+	/**
+	 * Returning the fields for the social media tab.
+	 *
+	 * @deprecated 14.9
+	 * @return array
+	 */
+	public function get() {
```

Comment:
To ensure backwards compatibility and avoid accidentally breaking something in an addon, we could temporarily call `WPSEO_Taxonomy_Fields::get('social')`.